Bottom: 789f02130cc6b3ec47fbfec01941834ef7f0918f
Top:    b8f1ecec09004dd7f26d6892339dc6fd1ce944ed
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-14 11:58:57 -0500

Refresh of all-docker_container

---
diff --git a/roles/docker-openssl/tasks/restore.yml b/roles/docker-openssl/tasks/restore.yml
index 0babd1d..21628c8 100644
--- a/roles/docker-openssl/tasks/restore.yml
+++ b/roles/docker-openssl/tasks/restore.yml
@@ -44,7 +44,7 @@
     image: '{{ openssl_image_name }}:{{ docker_image_tag }}'
     name: openssl_restore
     volumes:
-      - '{{ host_openssl_docker_restore_dir }}:{{ host_openssl_docker_restore_dir }}:z'
+      - '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}:{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}:z'
       - '{{ openssl_dv_name }}:/etc/ssl/private:z'
     command: ['extract', '{{ host_openssl_docker_restore_dir }}{{ openssl_backup_file }}']
